UI/reconciliation/report.html

```diff
@@ -260,6 +260,7 @@
     name = "action",
     text = text('Submit'),
     value = 'submit_recon_set',
+    disabled = ( their_total - beginning_balance != total_cleared_credits - total_cleared_debits ? 'true' : 'false'),
```

Comment:
you can use the variance for this value for this.  Where variance is 0, the report is balanced.